docs/apache-airflow/start/docker.rst

```diff
@@ -76,7 +76,7 @@ On **all operating system**, you need to run database migrations and create the
 
 .. code-block:: bash
 
-    docker-compose up --rm airflow-init
+    docker-compose up airflow-init
```

Comment:
```suggestion
    docker-compose run --rm airflow-init
```
This is an alternative command that better reflects the task we want to do.

Reply:
<img width="1270" alt="Screenshot 2021-01-26 at 16 03 38" src="https://user-images.githubusercontent.com/12058428/105862466-1429b700-5ff0-11eb-9563-37df20db7db0.png">